src/App.tsx

```diff
@@ -1,50 +1,120 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+
+type Props = {
+  initialGoods: string[];
+};
+
+enum SortType {
+  Alphabetical = 'Alphabetical',
+  ByLength = 'ByLength',
+  None = 'None',
+}
+
+function getPreparedGoods(
+  sortedGoods: string[],
+  currentSortType: SortType,
+  reversed: boolean,
+): string[] {
+  const updatedGoods = [...sortedGoods];
+
+  switch (currentSortType) {
+    case SortType.Alphabetical:
+      updatedGoods.sort((one, two) => one.localeCompare(two));
+      break;
+    case SortType.ByLength:
+      updatedGoods.sort((one, two) => one.length - two.length);
+      break;
+  }
+
+  if (reversed) {
+    updatedGoods.reverse();
+  }
+
+  return updatedGoods;
+}
+
+export const App: React.FC<Props> = ({ initialGoods }) => {
+  const [goods, setGoods] = useState(initialGoods);
```

Comment:
```suggestion

```
and you don't need the state for goods in this task

```
const sortedGoods = getPreparedGoods(initialGoods, type, isReversedActive);
```
just use sortedGoods everywhere